.github/workflows/publish.yml

```diff
@@ -253,6 +253,16 @@ jobs:
         ANDROID_KEYSTORE_PASSWORD: ${{ secrets.ANDROID_KEYSTORE_PASSWORD_CHIS_NE }}
         ANDROID_KEY_PASSWORD: ${{ secrets.ANDROID_KEY_PASSWORD_CHIS_NE }}
 
+    - name: Bundle cht_rci
```

Comment:
Unpack secrets, Assemble tasks missing?
Are you following [this](https://docs.communityhealthtoolkit.org/core/guides/android/branding/#:~:text=workflows/publish.yml.-,The%20Unpack%20secrets,-%E2%80%A6%20task%20unpacks%20and) guide?

Reply:
Ok, Added!